include/fiction/layouts/coordinates.hpp

```diff
@@ -665,6 +665,31 @@ struct formatter<fiction::cube::coord_t>
 };
 }  // namespace fmt
 
-#pragma GCC diagnostic pop
+template <typename CoordinateType>
+constexpr CoordinateType to_fiction_coord(const fiction::offset::ucoord_t& coord) noexcept
+{
+    return {coord.x, coord.y * 2 + coord.z};
+};
+/**
```

Comment:
Docstrings need to go above the function they are documenting. Otherwise, tools and developers alike will be very confused